readthedocs/restapi/views/model_views.py

```diff
@@ -243,7 +243,7 @@ class BuildViewSetBase(UserSelectViewSet):
     serializer_class = BuildSerializer
     admin_serializer_class = BuildAdminSerializer
     model = Build
-    filter_fields = ('project__slug',)
+    filter_fields = ('project__slug', 'commit')
```

Comment:
The commit has is not unique, so by querying only by commit you could receive 2 builds from different projects. I suppose that if the user cares about that, will also apply the filter `project__slug`.

I'm assuming that we are fine.